src/character.h

```diff
@@ -644,7 +648,7 @@ class Character : public Creature, public visitable<Character>
         bool male;
 
         std::list<item> worn;
-        std::array<int, num_hp_parts> hp_cur, hp_max;
+        std::array<int, num_hp_parts> hp_cur, hp_max, damage_bandaged, damage_disinfected;
```

Comment:
These new members are not saved / loaded anywhere.

Reply:
Fixed. It was tested without reloading.